File: M-MatrisÇarpma/MatrisCarpma.py
# İki matrisin boyutlarının girilmesi
print("A(m,n) Boyutlarini giriniz:")
m = int(input())
n = int(input())
print("B(f,p) Boyutlarini giriniz:")
f = int(input())
p = int(input())

# Çarpım koşulunun kontrol edilmesi
if n != f:
    print("Matrisler carpilamaz")
else:
    # Matrislerin oluşturulması
    A = [[0 for i in range(n)] for i in range(m)]
    B = [[0 for i in range(p)] for i in range(f)]
    C = [[0 for i in range(p)] for i in range(m)]

    # “A” matrisinin girilmesi
    print("A matrisini giriniz:")
    for i in range(m):
        for j in range(n):
            print('A[{}][{}]'.format(i+1, j+1))
            A[i][j] = int(input())
    
    # “B” matrisinin girilmesi
    print("B matrisini giriniz:")
    for i in range(f):
        for j in range(p):
            print('B[{}][{}]'.format(i+1, j+1))
            B[i][j] = int(input())
        
    # Çarpımın hesaplanması 
    for i in range(m):
        for j in range(p):
            for k in range(n):
                C[i][j] += A[i][k] * B[k][j];

print(C)

File: M-MatrisÇarpma/MatrisCarpma.cpp
#include <iostream>

using namespace std;

int main() {

    // Değişkenlerin tanımlanması.
    int m, n, f, p, i, j, k, ** A, ** B, ** C;
    cout << "A(m,n) Boyutlarini giriniz:";

    // İki matrisin boyutlarının girilmesi
    cin >> m;
    cin >> n;
    cout << "B(f,p) Boyutlarini giriniz:";
    cin >> f;
    cin >> p;
    
    // Matrislerin oluşturulması
    A = new int * [m];
    B = new int * [f];
    C = new int * [m];
    for (i = 0; i < m; i++)
        A[i] = new int[n];

    for (i = 0; i < f; i++)
        B[i] = new int[p];

    for (i = 0; i < m; i++)
        C[i] = new int[p];

    // Çarpım koşulunun kontrol edilmesi
    if (n != f)
        cout << "Matrisler carpilamaz";
    else {
        // A matrisinin elemenlarının girilmesi   
        for (i = 0; i < m; i++)
            for (j = 0; j < n; j++) {
                cout << "A[" << i + 1 << "][" << j + 1 << "]=";
                cin >> A[i][j];
            }
        cout << endl;

        // B matrisinin elemenlarının girilmesi
        for (i = 0; i < f; i++)
            for (j = 0; j < p; j++) {
                cout << "B[" << i + 1 << "][" << j + 1 << "]=";
                cin >> B[i][j];
            }

        // Çarpımın hesaplanması 
        for (i = 0; i < m; i++)
            for (j = 0; j < p; j++)
                for (k = 0; k < n; k++)
                    C[i][j] += A[i][k] * B[k][j];
        cout << "\nA*B=\n";

        // C matrisinin yazdırılması
        for (i = 0; i < m; i++) {
            for (j = 0; j < p; j++)
                cout << C[i][j] << "    ";
            cout << endl;
        }
    }
}